pyproject.toml

```diff
@@ -28,6 +31,27 @@ classifiers = [
     "License :: OSI Approved :: Apache Software License",
 ]
 
+[project.optional-dependencies]
+unit_tests = [
+    "ops==2.2.0",
+    "PyYAML==6.0",
+    "pytest==7.3.0",
+    "ops-scenario==2.1.3.4",
+]
+
+json_schemas = [
+    "PyYAML==6.0",
+]
+
+interface_tests = [
+    "PyYAML==6.0",
+    "pytest==7.3.0",
+    "ops-scenario==2.1.3.4",
```

Comment:
We should probably bump this

Reply:
This conversation was marked as resolved but the version wasn't bumped (it was only bumped in the `unit_test` section